src/main/java/az/gdg/authservice/security/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package az.gdg.authservice.security.controller;
+
+import az.gdg.authservice.security.dto.JwtAuthenticationRequest;
+import az.gdg.authservice.security.dto.JwtAuthenticationResponse;
+import az.gdg.authservice.security.service.AuthenticationService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/auth")
+@RestController
+@CrossOrigin
+@Api(value = "Authentication Controller")
+public class AuthenticationController {
+
+    private final AuthenticationService service;
+
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @ApiOperation("Create token if input credentials are valid")
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/signIn")
```

Comment:
Dash (or often called as hyphen) should be used to separate words in urls. 
Here are some examples of recomendations about it : 
[https://restfulapi.net/resource-naming/](url)
[https://nanvel.name/2016/02/rest](url)
Search for hyphen or dash on those webpages and it will come up.